Frends.SFTP.UploadFiles/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [2.7.1] - 2023-06-20
```

Comment:
You have added new parameters for the task, so you should increase the minor version instead of build version.